### DISCUSSION TITLE
Listening Debug And Trace Message

### DISCUSSION THREAD

**Discussion by chuongmep** [chuongmep/RevitAddInManager#23] [Ideas]
Will be create a tab allow listening event in revit addinmnager include : 
 - All Events
 - Trace Message
 - Debug Message
Sample : 
![image](https://user-images.githubusercontent.com/31106432/167344171-debe4d98-9637-41c6-8951-0bb6aa3606d2.png)

**Comment by quyet-arent3d** [comment]
@chuongmep 
Minh đang gặp vấn đề với cái add in manager
Bạn có thể chat trên linkedin ko nhỉ?
https://www.linkedin.com/in/quy%E1%BA%BFt-%E3%82%AF%E3%82%A8%E3%83%83%E3%83%88-465b06140/

**Reply by chuongmep** [reply]
Chào @quyet-arent, các vấn đề về lỗi với addin manager, bạn có thể đặt vấn đề tại https://github.com/chuongmep/RevitAddInManager/issues.

**Reply by quyet-arent3d** [reply]
@chuongmep Ngoài các vấn đề của addin manager thì mình thấy bạn pro nên muốn connect để học hỏi thêm. Nhờ bạn giúp đỡ.

**Reply by chuongmep** [reply]
Chào @quyet-arent, bạn có thể kết nối với mình qua linkedin, hiện tại mình đang trong thời gian đi xa nên không tiện trả lời, mình sẽ phản hồi sớm.Thân.

**Reply by quyet-arent3d** [reply]
@chuongmep Mình đã connect trên linkin chờ bạn trả lời. Cảm ơn nhiều nhé!

**Comment by Abdullah7977** [comment]
Hi I want to ask you question. how to make my revit view active (responsive) during  wpf window active ?

**Reply by chuongmep** [reply]
Hi @Abdullah7977, you research use class like this https://github.com/chuongmep/RevitAddInManager/blob/9c3cc0fbd80acffcd2c7ad65f3045fc4d2404744/AddInManager/Model/ProcessManager.cs#L17

**Reply by chuongmep** [reply]
https://github.com/chuongmep/RevitAddInManager/blob/9c3cc0fbd80acffcd2c7ad65f3045fc4d2404744/AddInManager/Command/AddinManagerBase.cs#L22

**Comment by Abdullah7977** [comment]
Thank you for your response and i will try it and tell you the result

**Reply by chuongmep** [reply]
Please note that need use ExternalEventHander in case you what execute some thing action ,https://www.revitapidocs.com/2023/f265a2c9-8540-9c97-9b37-4f7442becac2.htm
